src/main.cpp

```diff
@@ -3,13 +3,16 @@
 
 int main(int argc, char *argv[])
 {
-    qputenv("QT_SCALE_FACTOR", "2");
+    qputenv("QT_SCALE_FACTOR", "1");
```

Comment:
I think that it would be better to not have that line at all - it should be the user who choses the scale factor

Reply:
I will see how to do this